source/MapSalesPanel.cpp

```diff
@@ -225,13 +225,15 @@ int MapSalesPanel::CompareSpriteSwizzle() const
 void MapSalesPanel::DrawKey() const
 {
 	const Sprite *back = SpriteSet::Get("ui/sales key");
-	SpriteShader::Draw(back, Screen::TopLeft() + Point(WIDTH + 10, 0) + .5 * Point(back->Width(), back->Height()));
+	Point backPos = Screen::BottomLeft() + Point(WIDTH + 40., -back->Height());
+	SpriteShader::Draw(back, backPos + back->Center());
 
 	Color bright(.6f, .6f);
 	Color dim(.3f, .3f);
 	const Font &font = FontSet::Get(14);
 
-	Point pos(Screen::Left() + 50. + WIDTH, Screen::Top() + 12.);
+	Point padding(40., 48.);
```

Comment:
```suggestion
	static const Point padding(40., 48.);
```

Reply:
Should be capitalised probably

Reply:
I assumed the uppercase constants were `#define`d and the mixed-case ones were not.

Reply:
That is not the case in ES, all constants are CAPS, we don't use macros

Reply:
For my suggestions to take effect you will have to commit them.

Reply:
@RisingLeaf I changed it to PADDING. FWIW I created the padding variable because this is a simpler version of what I was working on where I moved the whole panel into interfaces.txt and that was the `pad` line. I expect to finish it at some point once I have figured out how to show the colours and buttons (the labels worked).